src/App/PropertyContainer.h

```diff
@@ -70,7 +70,7 @@ struct AppExport PropertyData
   };
 
   //purpose of this struct is to be constructible from all acceptable container types and to
-  //be able to return the offset to a property from the accepted containers. This allows to use
+  //be able to return the offset to a property from the accepted containers. This allows one to use
```

Comment:
This reads really strangely now, with the second occurrence of "one" in the sentence being numerical. In this particular case I might suggest using "you":
```suggestion
  //be able to return the offset to a property from the accepted containers. This allows you to use
```